CondCore/CondDB/src/OraDbSchema.cc

```diff
@@ -311,7 +311,7 @@ namespace cond {
 
     bool OraGTTable::select( const std::string& name ){
       cond::TagCollectionRetriever gtRetriever( m_session, "", "" );
-      return gtRetriever.existsTagCollection( name+"::All" );
+      return gtRetriever.existsTagCollection( name );
```

Comment:
This re-adds the need for CondDBv1 GT to have the `::All` in the name passed in the configuration